src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -812,7 +813,7 @@ private String createProgramGroupID(final SAMFileHeader header) {
                                                                                   final String name,
                                                                                   final Class<? extends Feature> featureType) {
 
-        final FeatureInput<? extends Feature> featureInput = new FeatureInput<>(name + ":" + filePath);
+        final FeatureInput<? extends Feature> featureInput = new FeatureInput<>(name + FEATURE_ARGUMENT_TAG_DELIMITER + filePath);
```

Comment:
Minor... Can you change this to `FeatureInput.FEATURE_ARGUMENT_TAG_DELIMITER` and change the import statement above?

Reply:
Sounds good.